src/cms/models/pages/page_translation.py

```diff
@@ -44,34 +42,12 @@ class PageTranslation(models.Model):
     language = models.ForeignKey(
         Language, related_name="page_translations", on_delete=models.CASCADE
     )
-    slug = models.SlugField(max_length=200, blank=True, allow_unicode=True)
-    title = models.CharField(max_length=250)
-    text = models.TextField(blank=True)
-    status = models.CharField(
-        max_length=6, choices=status.CHOICES, default=status.DRAFT
-    )
-    currently_in_translation = models.BooleanField(default=False)
-    version = models.PositiveIntegerField(default=0)
-    minor_edit = models.BooleanField(default=False)
     creator = models.ForeignKey(
         settings.AUTH_USER_MODEL,
         related_name="page_translations",
         null=True,
         on_delete=models.SET_NULL,
     )
-    created_date = models.DateTimeField(default=timezone.now)
-    last_updated = models.DateTimeField(auto_now=True)
-
-    @property
-    def foreign_object(self):
-        """
-        This property is an alias of the page foreign key and is needed to generalize the :mod:`~cms.utils.slug_utils`
-        for all content types
-
-        :return: The page to which the translation belongs
-        :rtype: ~cms.models.pages.page.Page
-        """
-        return self.page
 
     @property
     def ancestor_path(self):
```

Comment:
All tree and mirroring related functions and properties are not included in the AbstractBasePage and AbstractBasePageTranslation models and therefore not available for the ImprintPage and ImrpintPageTranslation model. Is this correct or does it have to be available there as well?

Examples:
* ancestor_path, combined_text for translation models
* depth, get_tree, get_mirrored_text, get_previous_sibling for page models